src/Microsoft.Framework.PackageManager/SourceControl/SourceCommand.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Framework.Runtime;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet;
+
+namespace Microsoft.Framework.PackageManager.SourceControl
+{
+    internal class SourceCommand
+    {
+        private const string SourcesFolderName = "sources";
+
+        private const string SourcesEnvironmentVariable = "DNX_SOURCES";
```

Comment:
I could agree with removing `CommonPrefix`, **especially** if all the environment variable names are in one place, because then it's just a matter of updating the prefix in one place.